build/csi-external-snapshotter/build.make.patch

```diff
@@ -0,0 +1,11 @@
+--- build.make	2020-05-23 16:43:42.192359292 +0100
```

Comment:
Was this file suppose to be checked in? OR perhaps it still seems to have a diff in the file output

Reply:
Aah indeed - I forgot to mention it! This project's Makefile happens to override the GOOS variable and seemed to always produce an `amd64` binary. Debugging a running build in `buildx` was tricky so this was the workaround I did locally a while ago. I presume there's something smarter we could try doing instead - Would you know where I could feed in [the BUILD_PLATFORMS variable](https://github.com/kubernetes-csi/external-snapshotter/blob/v2.1.1/release-tools/build.make#L70)? I think it just sets it to empty there.

The (builder) Dockerfile for this one installs `patch` and applies it to the checked out repo, before continuing on like the others.